### PR TITLE
Add memcached image.

### PR DESCRIPTION
**Pull request by kierenevans**
For Magento 1 sessions, and more!

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -57,6 +57,13 @@ services:
     depends_on:
       - varnish
 
+  memcached:
+    build:
+      context: ./memcached/1.4/
+    image: quay.io/continuouspipe/memcached1.4:v1.0
```
Comment (sroze) [inline docker-compose.yml]: Can we add a `-` between `memcached` and the version number?

Comment (tonypiper) [inline docker-compose.yml]: @sroze this looks consistent with the others...

Comment (sroze) [inline docker-compose.yml]: yeah it looks like... don't like this name tho... @andytson-inviqa what's your PoV?

Comment (tonypiper) [inline docker-compose.yml]: he's on leave until next year.

Comment (sroze) [inline docker-compose.yml]: @andytson-inviqa, are you back?

Comment (andytson-inviqa) [inline docker-compose.yml]: `quay.io/continuouspipe/memcached-1.4:v1.0` would prob be more preferable for me, but I don't think we should change it for just one image

Comment (andytson-inviqa) [inline docker-compose.yml]: I think I prefer more quay.io/continuouspipe/memcached:1.4-1.0 (which is more like how rpms and debs are done)

Comment (sroze) [inline docker-compose.yml]: In both cases that means that we need to rethink the entire image versioning to be consistent then... I agree with this PR and we can proceed with every image name in another PR then :)

**Pull request merged**